src/commands/env/delete/sandbox.ts

```diff
@@ -4,8 +4,14 @@
  * Licensed under the BSD 3-Clause license.
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
-import { Messages } from '@salesforce/core';
+import { Org, Messages } from '@salesforce/core';
 import { SfCommand, Flags } from '@salesforce/sf-plugins-core';
+import * as Interfaces from '@oclif/core/lib/interfaces';
+
+type FlagsDef = {
```

Comment:
I can see doing this for commands that choose to have a non-static `this.flags` property.  But why do it on the delete commands?

Reply:
Current version has the type removed.